src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -685,6 +686,16 @@ public class SimpleEvents {
 				.requiredPlugins("MC 1.16+");
 		}
 
+			Skript.registerEvent("Player Shear Entity", SimpleEvent.class, PlayerShearEntityEvent.class, "player (shear[s]|shearing) [an] entity")
```

Comment:
```suggestion
			Skript.registerEvent("Player Shear Entity", SimpleEvent.class, PlayerShearEntityEvent.class, "[player] shear[ing]")
```
This might be better? We should discuss syntax alternatives :)

Reply:
I suggested the syntax is to not confuse with the other 2 relevant events, [BlockShearEntityEvent](https://jd.papermc.io/paper/1.19/org/bukkit/event/block/BlockShearEntityEvent.html) and [PlayerShearBlockEvent](https://jd.papermc.io/paper/1.19/io/papermc/paper/event/block/PlayerShearBlockEvent.html). If any of these 2 events are implemented in the future, anyone who uses PlayerShearEntityEvent will be a breaking change.

Reply:
Why not make this PR add all three? They'd be relatively easy to combine into one shear event.

Reply:
Seems fair, as I haven’t thought of it while making this. I shall edit this PR and re-request review in a later time.